delfin/drivers/driver.py

```diff
@@ -115,3 +115,18 @@ def list_alerts(self, context, query_para=None):
     def clear_alert(self, context, sequence_number):
         """Clear alert from storage system."""
         pass
+
+    @abc.abstractmethod
```

Comment:
suggest remove @abc.abstractmethod, otherewise you need to add method for all drivers